ta4j-core/src/main/java/org/ta4j/core/num/PrecisionNum.java

```diff
@@ -744,7 +743,7 @@ public Num pow(Num n) {
         // use double pow(double, double)
         double xpowb = Math.pow(delegate.doubleValue(), bDouble);
         // use PrecisionNum.multiply(PrecisionNum)
-        BigDecimal result = xpowa.multiply(new BigDecimal(xpowb));
```

Comment:
I am not sure if replacing `new` with `valueOf ` does make sense here. Did you compare the results or why are you changing this? Using new or valueOf do not the same and in this case I can imagin that cutting off digits (that is what the valueOf method does) is not the preffered behaviour.

Reply:
Yes, you are right. Did not know that..